frontend/src/app/my-links/page.tsx

```
@@ -0,0 +1,196 @@
+          
+          <div className="relative w-full max-w-lg mb-3">
+            <div className="group bg-white dark:bg-gray-900 rounded-full flex items-center p-1 shadow-sm border border-gray-200 dark:border-gray-700 hover:border-purple-300 dark:hover:border-purple-700 transition-all duration-300">
+              <div className="pl-3 pr-2 text-gray-400 dark:text-gray-500 group-focus-within:text-purple-500 transition-colors flex items-center">
+                {/* <span className="material-symbols-outlined text-[18px]">auto_awesome</span> */}
+              </div>
+              <input 
+                className="w-full bg-transparent border-none focus:outline-none focus:ring-0 text-xs py-1.5 px-1 text-gray-800 dark:text-gray-100 placeholder-gray-400 font-normal" 
+                placeholder="Search folders or links" 
+                type="text"
+              />
+              <button className="flex items-center justify-center h-7 w-7 text-purple-600 hover:text-purple-700 hover:bg-purple-50 dark:hover:bg-purple-900/20 rounded-full transition-colors mr-0.5">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)